test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -2649,16 +2671,25 @@ public void PackCommand_PackSolution_AddsProjectRefsAsPackageRefs()
                     ProjectFileUtils.WriteXmlToFile(xml, stream);
                 }
 
+                // Act
                 msbuildFixture.RestoreSolution(testDirectory, solutionName, string.Empty);
 
-                // Act
-                msbuildFixture.PackSolution(testDirectory, solutionName, $"-o {testDirectory}");
+                // with custom output paths
```

Comment:
This one seems like an odd test to change. 
Can you elaborate why?

Reply:
I already have separate tests, but I wanted to get your opinion on this patch.

I want to test the scenario where I already built the project or solution with the defaults and then without cleaning it up, pack again into new custom path or vice versa.

Reply:
I'll separate those out, thanks.